frontend/src/metabase-lib/Question.ts

```diff
@@ -438,7 +450,8 @@ class QuestionInner {
     return this.setSettings({ ...this.settings(), ...settings });
   }
 
-  type(): string {
+  // TODO
+  private _type(): string {
```

Comment:
Not sure what's going on here?

Reply:
I deleted the TODO, it was left over from something else.